app/client/src/components/pages/LocationMap/mapStyles.css

```diff
@@ -147,7 +147,7 @@
   box-shadow: 0 1px 2px rgba(0, 0, 0, 0.3);
 }
 
-.hmw-map-toggle h4 {
+.hmw-map-toggle p {
```

Comment:
I changed this file in #108 to be an `<h1>` tag, so that the heading structure is still there. Can you revert this file?